docs/notebooks/atmodel_tutorial.ipynb

```diff
@@ -0,0 +1,1693 @@
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "ChristofKoch\n",
+      "Docs: [9, 221, 266, 272, 349, 411, 337, 371, 450, 483, 653, 663, 754, 712, 778, 921, 1212, 1285, 1254, 1533, 1489, 1580, 1441, 1657]\n",
+      "Topics:\n",
+      "[('Neuroscience', 0.99989393011046035)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "show_author('ChristofKoch')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Simple model evaluation methods\n",
+    "\n",
+    "We can compute the per-word bound, which is a measure of the model's predictive performance (you could also say that it is the reconstruction error).\n",
+    "\n",
+    "To do that, we need the `doc2author` dictionary, which we can build automatically."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from gensim.models import atmodel\n",
+    "doc2author = atmodel.construct_doc2author(model.corpus, model.author2doc)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's evaluate the per-word bound."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-6.9955968712\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Compute the per-word bound.\n",
+    "# Number of words in corpus.\n",
+    "corpus_words = sum(cnt for document in model.corpus for _, cnt in document)\n",
+    "\n",
+    "# Compute bound and divide by number of words.\n",
+    "perwordbound = model.bound(model.corpus, author2doc=model.author2doc, \\\n",
+    "                           doc2author=model.doc2author) / corpus_words\n",
+    "print(perwordbound)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can evaluate the quality of the topics by computing the topic coherence, as in the LDA class. Use this to e.g. find out which of the topics are poor quality, or as a metric for model selection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 15.6 s, sys: 4 ms, total: 15.6 s\n",
+      "Wall time: 15.6 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%time top_topics = model.top_topics(model.corpus)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Plotting the authors\n",
+    "\n",
+    "Now we're going to produce the kind of pacific archipelago looking plot below. The goal of this plot is to give you a way to explore the author-topic representation in an intuitive manner.\n",
```

Comment:
Please create a blog on wordpress linking to this ipynb with this blue graph as the title image.